docs/user-profile-and-api-key.md

```diff
@@ -0,0 +1,30 @@
+---
+id: user-profile-and-api-key
+title: User Profile & API Key
+---
+The ARGO User Profile page displays important user information including:
+- which email you have logged into the ARGO platform with
+- what your ARGO Member permission are
+- API Key management
+
+To get to your User Profile, click the initial icon in the top right corner and select Profile and Token.  
+## API Key
+Your authentication API Key uniquely identifies you, and your permissions, to all ARGO Data Platform applications, for example when using the Song or Score clients for data submission.
```

Comment:
Your authentication API token uniquely identifies you and your permissions. These permissions apply to all ARGO Data Platform applications, for example when using the Song or Score clients for molecular data submission.